src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -52,6 +52,27 @@ def load_checkpoint(checkpoint_path):
     for old_key, new_key in keys_to_rename.items():
         if old_key in sd:
             sd[new_key] = sd.pop(old_key)
+    
+    keys = list(sd.keys())
+    for key in keys:
+        if ".qkj_proj." in key:
+            value = sd[key]
+            # We split QKV in seperate Q,K,V
+
+            q_name = key.replace(".qkv_proj.",".q_proj.")
+            k_name = key.replace(".qkv_proj.", ".k_proj.")
+            v_name = key.replace(".qkv_proj.", ".v_proj.")
+
+            depth = value.shape[0]
+            assert depth % 3 == 0
+            # `SequeuceParallelTransformerBlock` has QKV weight is seperated in K,V,Q despite the naming:
```

Comment:
```suggestion
            # `SequenceParallelTransformerBlock` has QKV weight is separated in K,V,Q despite the naming:
```

Reply:
Actually there's a typo in `metaseq` so it really is `SequeuceParallelTransformerBlock`

Reply:
```suggestion
            # `SequeuceParallelTransformerBlock` has QKV weight is separated in K,V,Q despite the naming:
```